app/forms/webauthn_setup_form.rb

```diff
@@ -161,6 +160,14 @@ def mfa_user
     @mfa_user ||= MfaContext.new(user)
   end
 
+  def aaguid
+    begin
+      @aaguid ||= attestation_response&.authenticator_data&.aaguid
+    rescue StandardError
+      nil
+    end
+  end
```

Comment:
1. We can use a method-level rescue to simplify this.
2. I think we can be more targeted in the error we're looking for. Based on [where I suspect it's being raised](https://github.com/cedarcode/webauthn-ruby/blob/f030a78bd14999ce4905b74df92d8ca548832455/lib/webauthn/authenticator_data.rb#L33-L37), maybe we can rescue `WebAuthn::AuthenticatorDataFormatError` ?
3. Memoizing won't do much here if the value is `nil`, since `||=` will still evaluate for a `nil` value

```suggestion
  def aaguid
    attestation_response&.authenticator_data&.aaguid
  rescue WebAuthn::AuthenticatorDataFormatError
     nil
  end
```